tools/namespace.py

```diff
@@ -0,0 +1,32 @@
+import json
```

Comment:
For this internal, one time script, can you please put under support repo? There is a folder in there for one time scripts. 

Thanks.